meta-cli/src/deploy/actors/console.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
+// SPDX-License-Identifier: MPL-2.0
+
+use std::sync::Arc;
+
+use actix::prelude::*;
+
+use crate::config::Config;
+
+pub struct ConsoleActor {
+    #[allow(dead_code)]
+    config: Arc<Config>,
+}
+
+impl ConsoleActor {
+    pub fn new(config: Arc<Config>) -> Self {
+        Self { config }
+    }
+}
+
+impl Actor for ConsoleActor {
+    type Context = Context<Self>;
+
+    fn started(&mut self, _ctx: &mut Context<Self>) {
+        log::info!("console started");
+    }
+
+    fn stopped(&mut self, _ctx: &mut Context<Self>) {
+        log::info!("console stopped");
+    }
+}
+
+#[derive(Message)]
+#[rtype(result = "()")]
+pub struct Info(pub String);
+
+#[derive(Message)]
+#[rtype(result = "()")]
+pub struct Warning(pub String);
+
+#[derive(Message)]
+#[rtype(result = "()")]
+pub struct Error(pub String);
+
+impl Handler<Info> for ConsoleActor {
+    type Result = ();
+
+    fn handle(&mut self, msg: Info, _ctx: &mut Context<Self>) -> Self::Result {
+        log::info!("{}", msg.0);
+    }
+}
+
+impl Handler<Warning> for ConsoleActor {
+    type Result = ();
+
+    fn handle(&mut self, msg: Warning, _ctx: &mut Context<Self>) -> Self::Result {
+        log::warn!("{}", msg.0);
+    }
+}
+
+impl Handler<Error> for ConsoleActor {
+    type Result = ();
+
+    fn handle(&mut self, msg: Error, _ctx: &mut Context<Self>) -> Self::Result {
+        log::error!("{}", msg.0);
+    }
+}
+
+macro_rules! info {
```

Comment:
Maybe here a trait would be similar? Macros may be harder to grasp for people getting familiar with the project, thus I would keep them when really necessary. WDYT?